AddOns/Digipoort/dotnet.al

```diff
@@ -0,0 +1,47 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+dotnet
+{
+    assembly("Microsoft.Dynamics.NL.Digipoortservice")
+    {
+        Version = '15.0.0.0';
+        Culture = 'neutral';
+        PublicKeyToken = '31bf3856ad364e35';
+
+        type("Microsoft.Dynamics.NL.DigipoortServices"; "digipoortServices")
+        {
+        }
+
+        type("Digipoort.AanleverService.aanleverRequest"; "aanleverRequest")
```

Comment:
Even though the .Net classes are in Dutch, it's better to give them corresponding English names :)

Reply:
Hi Maria, these dotnet variables actually got implemented in BC16 base application (including the dutch names ;)), therefore I don't think the dotnet.al is needed anymore. I will remove it.